cfgov/unprocessed/css/organisms/video-player.less

```diff
@@ -103,6 +104,11 @@
     }
 }
 
+.o-video-player__loaded .o-video-player_play-btn {
+    opacity: 1;
```

Comment:
Updated in https://github.com/cfpb/cfgov-refresh/pull/5162/commits/868a228a15a0764b43eb6f4e9609c93693d39de6

I changed it to a link and button in the markup that are alternatively shown and hidden with the `u-hidden` class instead of using opacity.